### PR TITLE
chore(install): implement consistent output for recipe handling

### PR DESCRIPTION
**Pull request by zlesnr**

### CODE REVIEW

File: internal/install/recipe_installer.go [hunk 1]
```diff
@@ -58,8 +61,13 @@ const (
 )
 
 func (i *recipeInstaller) install() {
-	log.Infoln("Welcome to New Relic. Let's install some instrumentation.")
-	log.Infoln("Questions? Read more about our installation process at https://docs.newrelic.com/install-newrelic.")
+	fmt.Printf(`
+	Welcome to the New Relic. Let's install some instrumentation.
+
+	Questions? Read more about our installation process at
+	https://docs.newrelic.com/
```
Comment (zlesnr) [inline internal/install/recipe_installer.go:68]: Placeholder URL until someone says a smart thing.

File: internal/install/recipe_installer.go [hunk 2]
```diff
@@ -58,8 +61,13 @@ const (
 )
 
 func (i *recipeInstaller) install() {
-	log.Infoln("Welcome to New Relic. Let's install some instrumentation.")
-	log.Infoln("Questions? Read more about our installation process at https://docs.newrelic.com/install-newrelic.")
+	fmt.Printf(`
+	Welcome to the New Relic. Let's install some instrumentation.
```
Comment (ctrombley) [inline internal/install/recipe_installer.go:65]: `s/the New Relic/New Relic/`?

File: internal/install/recipe_installer.go [hunk 3]
```diff
@@ -102,16 +110,35 @@ func (i *recipeInstaller) install() {
...
 	}
 
-	log.Infoln("Success! Your data is available in New Relic.")
-	log.Infoln("Go to New Relic to confirm and start exploring your data.")
+	profile := credentials.DefaultProfile()
+	fmt.Printf(`
+	Success! Your data is available in New Relic.
+
+	Go to New Relic to confirm and start exploring your data.
+	https://one.newrelic.com/launcher/nrai.launcher?platform[accountId]=%d
```
Comment (ctrombley) [inline internal/install/recipe_installer.go:118]: 👍

File: internal/install/recipe_installer.go [hunk 4]
```diff
@@ -102,16 +110,35 @@ func (i *recipeInstaller) install() {
...
+	`, profile.AccountID)
+
+	fmt.Println()
 }
 
 func (i *recipeInstaller) discoverFatal() *discoveryManifest {
+	s := spinner.New(spinner.CharSets[14], 100*time.Millisecond)
```
Comment (ctrombley) [inline internal/install/recipe_installer.go:125]: We can consider abstracting this later on maybe?

File: internal/install/recipe_installer.go [hunk 5]
```diff
@@ -170,11 +197,23 @@ func (i *recipeInstaller) installLoggingFatal(m *discoveryManifest, recipes []re
 }
 
 func (i *recipeInstaller) fetchRecommendationsFatal(m *discoveryManifest) []recipe {
+	s := spinner.New(spinner.CharSets[14], 100*time.Millisecond)
```
Comment (ctrombley) [inline internal/install/recipe_installer.go:200]: We can consider abstracting this later on maybe?

File: internal/install/recipe_installer.go [hunk 6]
```diff
@@ -102,16 +110,35 @@ func (i *recipeInstaller) install() {
...
+
+	s.Start()
+	defer func() {
+		s.Stop()
+		fmt.Println(s.Suffix)
+	}()
+
 	m, err := i.discoverer.discover(utils.SignalCtx)
 	if err != nil {
+		s.FinalMSG = "\u1F4A5"
```
Comment (ctrombley) [inline internal/install/recipe_installer.go:136]: Hoist to package-level const?

File: internal/install/recipe_installer.go [hunk 7]
```diff
@@ -102,16 +110,35 @@ func (i *recipeInstaller) install() {
...
 		log.Fatalf("Could not install New Relic.  There was an error discovering system info: %s", err)
 	}
 
+	s.FinalMSG = "\u2705"
```
Comment (ctrombley) [inline internal/install/recipe_installer.go:140]: Hoist to package-level const?

File: internal/install/recipe_installer.go [hunk 8]
```diff
@@ -170,11 +197,23 @@ func (i *recipeInstaller) installLoggingFatal(m *discoveryManifest, recipes []re
...
+
+	s.Start()
+	defer func() {
+		s.Stop()
+		fmt.Println(s.Suffix)
+	}()
+
 	recipes, err := i.recipeFetcher.fetchRecommendations(utils.SignalCtx, m)
 	if err != nil {
+		s.FinalMSG = "\u1F4A5"
```
Comment (ctrombley) [inline internal/install/recipe_installer.go:211]: Hoist to package-level const?

File: internal/install/recipe_installer.go [hunk 9]
```diff
@@ -170,11 +197,23 @@ func (i *recipeInstaller) installLoggingFatal(m *discoveryManifest, recipes []re
...
 		log.Fatalf("Could not install New Relic. Error retrieving recipe recommendations: %s", err)
 	}
 
+	s.FinalMSG = "\u2705"
```
Comment (ctrombley) [inline internal/install/recipe_installer.go:215]: Hoist to package-level const?

File: internal/install/recipe_installer.go [hunk 10]
```diff
@@ -265,14 +298,27 @@ func (i *recipeInstaller) reportRecipeFailed(e recipeStatusEvent) {
...
+
+	s.Start()
+	defer func() {
+		s.Stop()
+		fmt.Println(s.Suffix)
+	}()
+
 	ok, err := i.executeAndValidate(m, r)
 	if err != nil {
+		s.FinalMSG = "\u1F4A5"
```
Comment (ctrombley) [inline internal/install/recipe_installer.go:312]: Hoist to package-level const?

File: internal/install/recipe_installer.go [hunk 11]
```diff
@@ -265,14 +298,27 @@ func (i *recipeInstaller) reportRecipeFailed(e recipeStatusEvent) {
...
 		log.Fatalf("Could not install %s: %s", r.Name, err)
 	}
 
 	if !ok {
+		s.FinalMSG = "\u1F4A5"
```
Comment (ctrombley) [inline internal/install/recipe_installer.go:317]: Hoist to package-level const?

File: internal/install/recipe_installer.go [hunk 12]
```diff
@@ -265,14 +298,27 @@ func (i *recipeInstaller) reportRecipeFailed(e recipeStatusEvent) {
...
 		log.Fatalf("Could not detect data from %s.", r.Name)
 	}
+
+	s.FinalMSG = "\u2705"
```
Comment (ctrombley) [inline internal/install/recipe_installer.go:321]: Hoist to package-level const?

File: internal/install/recipe_installer.go [hunk 13]
```diff
@@ -265,14 +298,27 @@ func (i *recipeInstaller) reportRecipeFailed(e recipeStatusEvent) {
 }
 
 func (i *recipeInstaller) executeAndValidateFatal(m *discoveryManifest, r *recipe) {
+	s := spinner.New(spinner.CharSets[14], 100*time.Millisecond)
```
Comment (ctrombley) [inline internal/install/recipe_installer.go:301]: We can consider abstracting this later on maybe?

**Review by ctrombley — APPROVED** [review]
🎉

**Comment by ctrombley** [comment]
@zlesnr looks great AND great branch name to boot.  Approved with a few suggestions.

File: internal/install/recipe_installer.go (hunk 4 above)
Comment (zlesnr) [reply internal/install/recipe_installer.go:125]: Yeah, I think to get a standard, we should probably move that out to a method.

File: internal/install/recipe_installer.go (hunk 6 above)
Comment (zlesnr) [reply internal/install/recipe_installer.go:136]: Yeah, good call.

**Pull request merged**